earth_enterprise/rpms/build.gradle

```diff
@@ -55,8 +55,16 @@ def openGeeVersion =
         it // The first string that remains is the version string.
     }
 
+def stagedInstall_path = '/tmp/fusion_os_install'
+if (new File('build.properties').exists()) {
```

Comment:
Minor: Why instantiate this file object more than once instead of assigning it to a variable. Probably minuscule as far as performance, just curious if this is a preferred style for Gradle or something like that.